docs/design/adminconsole/INSTANCE-DATA.md

```diff
@@ -142,6 +142,46 @@ Also supports `GET /adminconsole/odsInstances/{id}`
   }
   ```
 
+> [!IMPORTANT]
+> The following diagram represents the possible values the Instances 
+> have during the Worker's process
+```mermaid
```

Comment:
Please add an extra line break above this for better readability of the raw markdown.